test/wkrep/wkb_parser_test.rb

```diff
@@ -127,7 +130,10 @@ def test_linestring_basic
   def test_linestring_with_ewkb_z
     factory_ = RGeo::Cartesian.preferred_factory(has_z_coordinate: true)
     parser = RGeo::WKRep::WKBParser.new(factory_, support_ewkb: true)
-    obj = parser.parse("0080000002000000023ff000000000000040000000000000004008000000000000401000000000000040140000000000004018000000000000")
+    obj = parser.parse(<<~TEXT.split.join)
```

Comment:
```suggestion
    obj = parser.parse("xxxx" \
     "xxxx" \
     "xxx")
```
I think I'd prefer this form as it is more efficient, and less obscure to newer people to ruby

Reply:
Would you like me to do this for all of the heredocs that I have done for this?

Reply:
@thestelz Yes I think so.

There is only one exception I have in mind. I'd use heredoc only for text that you want to keep multiline. So for instance, you could still use heredoc for a wkt if you feel it is clearer:

```ruby
<<~WKT
GEOMETRYCOLLECTION(
  LINESTRING(1 1, 2 2),
  ...
)
WKT

```